index.html

```diff
@@ -0,0 +1,345 @@
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <footer>
+        <section class="form-section">
+          <h3 class="form-heading">
+            I'm always interested in hearing about new projects, so if you would
+            like to chat, please get in
+          </h3>
+          <form
+            action="https://formspree.io/f/mqkwlyzj"
+            class="flex column align-center"
+            method="POST"
+            enctype="multipart/form-data"
+          >
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="Full name"
+              maxlength="30"
+              required
+            />
+            <input
+              type="text"
+              name="email"
+              id="email"
+              placeholder="Email address"
+            />
+            <textarea
+              name="message"
+              id="message"
+              cols="30"
+              rows="5"
+              placeholder="Enter text here"
+              maxlength="500"
+            ></textarea>
+            <input type="submit" value="Get in touch" />
+          </form>
+        </section>
+        <section>
+          <img src="media/svg/github.svg" alt="" />
```

Comment:
```suggestion
          <img src="media/svg/github.svg" alt="" />
```
```suggestion
          <img src="media/svg/github.svg" alt="github social media logo" />
```
[OPTIONAL]
- Please always add alt text to images so that it can help screen readers to read for improving the accessibility and inclusion of the website.